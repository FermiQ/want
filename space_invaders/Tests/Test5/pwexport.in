&inputpp
  prefix='gold1'
  outdir='./'
  pseudo_dir = './'
  psfile(1) = 'Au.pw91-d-van.UPF'
  single_file = .FALSE.
  ascii = .FALSE.
/
