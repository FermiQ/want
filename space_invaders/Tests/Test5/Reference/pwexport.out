
     Program POST-PROC v.2.1cvs starts ...
     Today is 18Mar2005 at 11:59:14 

     Reading file gold1.save ...      only dimensions
     read complete

     Reading file gold1.save ...      all except wavefuctions
     read complete

     nbndx  =    10  nbnd   =    10  natomwfc =     9  npwx   =    2911
     nelec  =   11.00 nkb   =    18  ngl    =    4885
